lib/modules/hosts/imgur.js

```diff
@@ -219,7 +219,6 @@ const imgur = {
 			type: 'VIDEO',
 			controls: false,
 			href: ShowImages.module.options.preferredImgurLink.value === 'share' ? `${imgur.baseURL}${info.id}` : `${info.link}`,
-			downloadurl: info.download,
```

Comment:
Note: the reason imgur stopped working is because they (seemingly) stopped sending a `download` url; this is always undefined.